packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,42 @@
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Box } from '../../Box';
+import { Crumb } from './Crumb';
+import { CrumbLink } from './CrumbLink';
+import { CrumbSimpleMenu } from './CrumbSimpleMenu';
+import { Divider } from './Divider';
+import { Flex } from '../../Flex';
+
+const ListFlex = styled.ol`
+  display: flex;
+`;
+
+export const Breadcrumbs = ({ label, children, ...props }) => {
+  return (
+    <Box aria-label={label} {...props}>
+      <ListFlex aria-hidden={true}>
+        {Children.map(children, (child, index) => {
+          const isLast = index + 1 === children.length;
+
+          return (
+            <Flex inline as="li">
+              {child}
+              {!isLast && <Divider />}
```

Comment:
@HichamELBSI This approach should make it fairly easy to make the divider a prop once it is needed.

Reply:
We've switched to use slashes in e9e1036c4c2fa601ee99119cbd13775d31b81100